jwst/gain_scale/tests/test_gain_scale.py

```diff
@@ -0,0 +1,62 @@
+"""
+Unit tests for gain_scale correction
+"""
+
+from jwst.datamodels import RampModel
+from jwst.gain_scale.gain_scale import do_correction
+from jwst.gain_scale import GainScaleStep
+import numpy as np
+import pytest
+
+def test_correction(make_rampmodel):
+    """Make sure correct gain factor is applied
+    """
+    datmod = make_rampmodel(2, 2048, 2048)
```

Comment:
I have no idea how long this test takes to run, but it could be made faster by not using a full 2048x2048 data array. You don't really need to test that many pixels to confirm that the math is being applied properly. Besides, in real life, the gain_scale step only gets applied to subarray data anyway. I would think that something on the order of 50x50 or 200x200 would be sufficient.

Reply:
Reduced size to 50x50.  Tests run very fast.  Now they run faster.